.cirrus.yml

```diff
@@ -89,3 +89,18 @@ multi_arch_test_docker_builder:
     - echo "Test the ${STAGING_IMAGE_NAME}:${CURRENT_TAG} image supporting linux/${CIRRUS_ARCH}"
     - TEST_IMAGE="${STAGING_IMAGE_NAME}:${CURRENT_TAG}" bats --tap test
   depends_on: multi_arch_build
+
+sonar_scan_task:
```

Comment:
I'm looking at [Next and I don't see the Dockerfile anymore](https://next.sonarqube.com/sonarqube/code?id=SonarSource_sonar-scanner-cli-docker_AYc24KKjsMmHG0qPXLZ9&pullRequest=234) on the code tab.

Reply:
Maybe because we only show changed files on a PR analysis?

Reply:
https://cirrus-ci.com/task/5346807927013376?logs=main#L91 seems to prove the Docker file was indexed by the scanner

Reply:
Ah yes, indeed